### PR TITLE
feat: Update virtualenv recommended practices

### PR DESCRIPTION
**Pull request by KingMichaelPark**

### CODE REVIEW

File: standards/python-standards.md [hunk 1]
```diff
@@ -49,14 +49,78 @@ managing multiple Python versions for different projects.
...
+In order to have it subsequently auto-activate when you change directories to it
+and deactivate when you leave it then you can use the following in your
+equivalent `~/.bashrc` or `~/.zshrc` files
+
+For Bash Users:
+
+```bash
+alias dovenv="python -m venv .venv && source ./.venv/Scripts/activate"
+alias venva="source ./.venv/Scripts/activate"
+cd() {
```
Comment (gvee-uk) [inline standards/python-standards.md:92]: Improved script that better follows your approach (I forgot about the deactivate!)

```bash
cd() {
  builtin cd "$1"

  MYENV="./.venv"

  # When you cd into a folder that contains $MYENV
  [[ -d $MYENV ]] && [[ ! $VIRTUAL_ENV ]] && venva

  # When you cd into a folder that doesn't have
  [[ ! -d $MYENV ]] && [[ $VIRTUAL_ENV ]] && deactivate
}
```

**Review by gvee-uk — CHANGES_REQUESTED** [review]

File: standards/python-standards.md (hunk 1 above)
Comment (gvee-uk) [reply standards/python-standards.md:92]: Further updated to use functions instead of aliases!

Comment (gvee-uk) [reply standards/python-standards.md:92]: Another minor tweak (sorry!). Functions > aliases. Share the `MYENV` across all functions

**Comment by PsypherPunk** [comment]
I don't think this is the right way to do this: if our standards are "here's a script you can run", those aren't standards.

`pyenv-virtualenv` was collectively chosen at the time as the only tool that met our requirements. And, based on this, it still does: we require using `pyenv-installer` which installs both `pyenv` and `pyenv-virtualenv` and, at least for *nix machines, does everything herein. If that's not being done, it renders this document—and any updates—pointless anyway.

If we want to change this, can we have a more considered approach? Replacing this with a better tool would be ideal but it should ideally be just that: a tool, even if it's one we develop ourselves.

Alternatively, if we can't find a solution: drop the section entirely. Requiring that people use virtual environments can be documented herein but specifics around scripts, tools, kludges, etc. can be moved elsewhere.

**Pull request closed without merging**